Extensions/Effects/JsExtension.js

```diff
@@ -887,7 +905,7 @@ module.exports = {
       new gd.PropertyDescriptor(/* defaultValue= */ '1.0')
         .setLabel(_('Vignetting Alpha (between 0 and 1)'))
         .setType('number')
-        .setDescription('The size of the noise particles')
+        .setDescription('Amount of opacity of vignette')
```

Comment:
Actually we can remove the setDescription entirely, this is not giving much information.

Reply:
Thank you and how about `Vignetting Blur`'s description? I think it can be removed too.

Reply:
Yes we can remove it too I think! :)